rafthttp/stream.go

```diff
@@ -38,6 +39,16 @@ const (
 	streamBufSize = 4096
 )
 
+var (
+	errUnsupportedStreamType = fmt.Errorf("unsupported stream type")
+
+	streamSupportMap = map[streamType]*semver.Version{
```

Comment:
i do not think this will work in the long term.

2.0.0 means all version larger than 2.0.0 should support this? what if we want to deprecate one? 

a more easy way is to make the version as the key.